Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = () => {
+    return (
+        <div class= "header-title">
+            <h1>Lambda School @LambdaSchool * 26 jan</h1>
```

Comment:
This definitely works for the h1. consider breaking this down further.
we can have our h1 element nested inside of a div and we can have Lambda school as an h1
and have `@LambdaSchool` inside of a span and our date inside of a span. so If we wanted to include functionality like having our `@LambdaSchool` to be clickable and our date to be dynamic we can easily do that with our new structure.